src/panel/pages/timeline/Timeline.tsx

```diff
@@ -1,27 +1,48 @@
 import React, { FC } from "react";
 import styled from "styled-components";
 import { useTimelineContext } from "../../context";
-import { TimelineRow } from "./components";
+import { Background } from "../../components/Background";
+import { TimelineRow, TimelinePane } from "./components";
 
 export const Timeline: FC = () => {
-  const { setContainer, getTimePosition, events } = useTimelineContext();
+  const {
+    setContainer,
+    getTimePosition,
+    events,
+    selectedEvent
+  } = useTimelineContext();
+
+  // console.log(events);
 
   // We lie about the types to save having to do this check
   // in every component. This guard is needed.
-  if (!getTimePosition) return <Container ref={setContainer} />;
+  if (!getTimePosition)
```

Comment:
is it possible that this function will be undefined or did you forget to call it?

Reply:
It's possible the function will be undefined afaik. I mainly just wanted to add the wrapper